news_items/012_nested_questions/index.html

```diff
@@ -0,0 +1,42 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ .code-block {
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     width: 70%;
+     margin-left: auto;
+     margin-right: auto;
+ }
+ .code-block > pre {
+     padding: 0.5rem;
+     display: block;
+     margin-bottom: 0px;
+     line-height: 125%;
+     overflow: wrap;
+ }
+</style>
+
+<p>
+    <a href="https://github.com/nicknytko">Nicolas Nytko</a> (CS BS '19, CS MS '22)
+</p>
+
+<p>
+    Questions now have the ability to be nested inside of subfolders &mdash; meaning that instead of a sprawling <code class="user-output">questions</code> folder inside of your course, you can now organize these into folders however you wish!
+</p>
+
+<img src="nested.png" width="875px">
+
+<p>
+    The full QID of the question is now the path from the <code class="user-output">questions</code> folder.  For example, the <code class="user-output">demoAnsiOutput</code> folder above would be <code class="user-output">demoQuestions/demoAnsiOutput</code> when creating assessments.
+</p>
```

Comment:
Could we add a screenshot showing what a few rows of the questions page looks like with the above subfolders? And/or show a snippet of an `infoAssessment.json` with these news QIDs containing slashes?